langchain/retrievers/bibtex.py

```diff
@@ -0,0 +1,18 @@
+from typing import List
+
+from langchain.schema import BaseRetriever, Document
+from langchain.utilities.bibtex import BibtexparserWrapper
+
+
+class BibtexRetriever(BaseRetriever, BibtexparserWrapper):
+    """
+    It is effectively a wrapper for BibtexparserWrapper.
+    It wraps load() to get_relevant_documents().
+    It uses all BibtexparserWrapper arguments without any change.
+    """
+
+    def get_relevant_documents(self, file_path: str) -> List[Document]:
```

Comment:
get_relevant_documents should only take in a query (shouldn't change the BaseRetriever interface). see retrievers/arxiv.py for an example

Reply:
if we don't have a good way of searching over bibtex we can remove the Retriever for now, just a document loader is already nice

Reply:
Fair point, I removed the retriever.